examples/cluster/plot_kmeans_digits.py

```diff
@@ -53,14 +53,13 @@
 
 
 print(79 * '_')
-print('% 9s' % 'init'
-      '    time  inertia    homo   compl  v-meas     ARI AMI  silhouette')
+print('%9s' % 'init\t\ttime\tinertia\thomo\tcompl\tv-meas\tARI\tAMI\tsilhouette')
```

Comment:
there are two `\t` after init, I don't think that would line up well. also this line seems too long for PEP8 (though i believe removing the duplicated `\t` if it was unintentional would just put it at 79 chars)